code/modules/antagonists/heretic/magic/rust_charge.dm

```diff
@@ -5,36 +5,48 @@
 		will deal high damage to others and rust around you during the charge. \
 		As it is the rust that empowers you with this ability, no focus is needed."
 	charge_distance = 10
-	charge_damage = 50
+	charge_damage = 25
 	cooldown_time = 45 SECONDS
+	charge_past = 0
 
 /datum/action/cooldown/mob_cooldown/charge/rust/Activate(atom/target_atom)
 	var/turf/open/start_turf = get_turf(owner)
 	if(!istype(start_turf) || !HAS_TRAIT(start_turf, TRAIT_RUSTY))
 		return FALSE
 	StartCooldown(135 SECONDS, 135 SECONDS)
+	if(ishuman(owner))
+		var/mob/living/carbon/human/human_owner = owner
+		human_owner.physiology.damage_resistance += 100
+	RegisterSignal(owner, COMSIG_FINISHED_CHARGE, PROC_REF(affect_aoe))
+	var/mob/living/carbon/carbon_owner = owner
+	carbon_owner.uncuff()
+	var/obj/item/clothing/shoes/shoes = carbon_owner.shoes
+	if(istype(shoes) && shoes.tied == SHOES_KNOTTED)
+		shoes.adjust_laces(SHOES_TIED, carbon_owner)
```

Comment:
I also think this is lame and powercreepy

Reply:
ok it's been removed